Update file1.adoc (#12)
alxlenc/pygh-tests: content/file1.adoc

## This is a Title
I create a file.

> When a cluster is provisioned, each machine in the cluster requires information about the cluster.

Reviewer: How about...
```suggestion
Each machine in the cluster needs information about the cluster once it has been provisioned.
```
